### PR TITLE
Product Section work

### PR DESCRIPTION
**Pull request by mdzzg**
- Finalized my part of the **Data Analysis & Validation**. No further modifications forseen from my side in this very section.

- Modified the **About the OET Product** section, as well as the **Product Principles** section, which _needs further work_, such they align, do not overlap, and hold contradictory info. _Which is the case ATM - hence not finished here just yet._

- Merged **Product Principles Flow** & **Product Principles Timeline** into one section titled **Product Principles**, keeping the respective sub-headings.

`Kingly accept this constructive PR`

### CODE REVIEW

File: docs/Product/AboutTheOETProduct.mdx [hunk 1]
```diff
@@ -6,25 +6,27 @@ sidebar_label: "About the OET Product"
...
+Open Energy Transition continues to build on open source software. PyPSA project, initially started in Academia, back in 2016, is being developed actively.
 
-## Sub Heading
+As of now, the PyPSA engine counts approx. 25,000 lines of Python code.
 
-At our company, we are committed to providing a safe and inclusive workplace for all employees. This policy outlines our expectations for behavior and conduct in the workplace.
+## Product Development
 
-## Another Sub Heading
+Open Energy Transition actively uses, and extend PyPSA, for the purposes of studies, consultations, workshops, presentations, conferences and projects.
```
Comment (finozzifa) [inline docs/Product/AboutTheOETProduct.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

extend --> extends

File: docs/Product/AboutTheOETProduct.mdx [hunk 2]
```diff
@@ -6,25 +6,27 @@ sidebar_label: "About the OET Product"
...
 
-All employees are expected to adhere to the following code of conduct:
+- Any such modifications are backfed upstream, to the original PyPSA-X repository.
 
-- Treat all colleagues with respect and professionalism
-- Avoid any behavior that could be considered discriminatory or harassing
-- Maintain confidentiality of company and client information
-- Follow all company policies and procedures
-- Report any violations of this code of conduct to management
+The code gets reviewed & modified by dozens of associates/contributors in power systems planning, operation & energy economics, many of whom are experts/PhDs in their specific areas.
```
Comment (finozzifa) [inline docs/Product/AboutTheOETProduct.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

& --> and

Comment (finozzifa) [inline docs/Product/AboutTheOETProduct.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Please remove _experts/PhDs_. I am not an expert in any area or topic related to power energy modelling, yet OET welcomes my contributions.

We do not want to scare people off :)

File: docs/Product/DataAnalysisValidation.mdx [hunk 3]
```diff
@@ -1,27 +1,29 @@
 ---
 sidebar_position: 10.11
-sidebar_label: "Data analysis and validation"
+sidebar_label: "Data Analysis and Validation"
```
Comment (finozzifa) [inline docs/Product/DataAnalysisValidation.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Please leave Data analysis and validation. As highlighted here, we should use _sentence case_ for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: docs/Product/DataAnalysisValidation.mdx [hunk 4]
```diff
@@ -1,27 +1,29 @@
...
 ---
 
 import MaintainerTag from "@site/src/components/MaintainerTag";
 import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
 
 <MaintainerTag maintainerEmails={["mario.dzamarija@openenergytransition.org", "fabrizio.finozzi@openenergytransition.org"]} />
 
-# Data analysis and validation
+# Data Analysis and Validation
```
Comment (finozzifa) [inline docs/Product/DataAnalysisValidation.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Please leave Data analysis and validation. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: docs/Product/DataAnalysisValidation.mdx [hunk 5]
```diff
@@ -1,27 +1,29 @@
...
 <ControlledDocBanner />
 
-## Data analysis
+## Data Analysis
```
Comment (finozzifa) [inline docs/Product/DataAnalysisValidation.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Please leave Data analysis. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: docs/Product/DataAnalysisValidation.mdx [hunk 6]
```diff
@@ -1,27 +1,29 @@
...
-Data analysis performed by us revolves around gathering the data required for modeling the power systems, from various aspects.
+Data analysis performed by us revolves around gathering the data, required for modeling the power systems, in various aspects.
 
-- It is both the technical and the market data, utilized as a backbone for the power system expansion, unit commitment/economic dispatch, and the power flow calculation.
-- Data analysis plays a crucial role, prior the computer simulation, as the correct input data is the foundation for the work that follows.
+- It is both the technical and the market data, used as the backbone for the power system expansion, unit commitment/economic dispatch, and the power flow simulation.
+- Data analysis has a crucial role, prior the computer simulation, as the correct input data is the foundation of the work that follows.
 
-## Data validation
+## Data Validation
```
Comment (finozzifa) [inline docs/Product/DataAnalysisValidation.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Please leave Data validation. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: docs/Product/ProductDevelopment.mdx [hunk 7]
```diff
@@ -1,30 +1,30 @@
 ---
 sidebar_position: 10.7
-sidebar_label: "Product Development Flow"
+sidebar_label: "Product Development"
```
Comment (finozzifa) [inline docs/Product/ProductDevelopment.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Product development. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: docs/Product/ProductDevelopment.mdx [hunk 8]
```diff
@@ -1,30 +1,30 @@
...
 ---
 
 import MaintainerTag from "@site/src/components/MaintainerTag";
 import ControlledDocBanner from "@site/src/components/ControlledDocBanner";
 
 <MaintainerTag maintainerEmails={["jane.smith@example.com", "john.doe@example.com"]} />
 
-# Product Development Flow
+# Product Development
```
Comment (finozzifa) [inline docs/Product/ProductDevelopment.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Product development. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: docs/Product/ProductDevelopment.mdx [hunk 9]
```diff
@@ -1,30 +1,30 @@
...
 this should be same as sidebar_label and folder name
 
 <ControlledDocBanner />
 
 should be present on every controlled document.
 
-## Sub Heading
+## Product Development Flow
```
Comment (finozzifa) [inline docs/Product/ProductDevelopment.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Product development flow. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: docs/Product/ProductDevelopment.mdx [hunk 10]
```diff
@@ -1,30 +1,30 @@
...
 
 At our company, we are committed to providing a safe and inclusive workplace for all employees. This policy outlines our expectations for behavior and conduct in the workplace.
 
-## Another Sub Heading
+## Product Development Timeline
```
Comment (finozzifa) [inline docs/Product/ProductDevelopment.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

Product development timeline. As highlighted here, we should use sentence case for titles https://open-energy-transition.github.io/handbook/docs/Handbook/HandbookStyleGuide

File: handbook/docs/Product/DataAnalysis.mdx [hunk 11]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (finozzifa) [inline handbook/docs/Product/DataAnalysis.mdx]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

I think that the information present in this file is contained already in DataAnalysisValidation.mdx. I would therefore delete DataAnalysis.mdx

File: package-lock.json [hunk 12]
```diff
@@ -7352,11 +7352,11 @@
       "integrity": "sha512-8+9WqebbFzpX9OR+Wa6O29asIogeRMzcGtAINdpMHHyAg10f05aSFVBbcEqGf/PXw1EjAZ+q2/bEBg3DvurK3Q=="
```
Comment (finozzifa) [inline package-lock.json]: Dear Mario thanks a lot for your kind contribution. Please find below my suggestion

If you could, please revert your changes to package-lock.json. Thanks :)

**Comment by finozzifa** [comment]
Hi @mdzzg ,
thanks a lot for your pull request and contributions. 
I have left a few comments here and around. 

Please remember that, just for the sake of brevity, I have not repeated in all my comments the following intro

_Dear Mario thanks a lot for your kind contribution. Please find below my suggestion_

File: package-lock.json
Comment (drifter089) [inline package-lock.json:1]: Hey Mario,
please remove package-lock.json file from this PR.
we have updated the .gitignore file in this repository so this doesn't get pushed in future.
please also update you .gitignore so package-lock.json is ignored automatically.

File: docs/Product/AboutTheOETProduct.mdx (hunk 1 above)
Comment (mdzzg) [reply docs/Product/AboutTheOETProduct.mdx]: Hey Fabrizio! Thanks a lot for your comments.

I have changed the 'extend' to 'extends'

File: docs/Product/AboutTheOETProduct.mdx (hunk 2 above)
Comment (mdzzg) [reply docs/Product/AboutTheOETProduct.mdx]: Amperesign & is now removed from the sentences in the Product section.

Comment (mdzzg) [reply docs/Product/AboutTheOETProduct.mdx]: You are right. I have not thought about this, and I modified, as you have done previously.

Please check the new sentence version in the modified commit of this PR.

File: docs/Product/DataAnalysisValidation.mdx (hunk 3 above)
Comment (mdzzg) [reply docs/Product/DataAnalysisValidation.mdx]: I corrected the sentence case for the Anaysis & validation.

Also for the OET Product, which is now also writtent in the sentence case - OET product.

File: docs/Product/DataAnalysisValidation.mdx (hunk 4 above)
Comment (mdzzg) [reply docs/Product/DataAnalysisValidation.mdx]: Corrected.

File: docs/Product/DataAnalysisValidation.mdx (hunk 5 above)
Comment (mdzzg) [reply docs/Product/DataAnalysisValidation.mdx]: Also corrected.

File: docs/Product/DataAnalysisValidation.mdx (hunk 6 above)
Comment (mdzzg) [reply docs/Product/DataAnalysisValidation.mdx]: The same - corrected.

File: docs/Product/ProductDevelopment.mdx (hunk 7 above)
Comment (mdzzg) [reply docs/Product/ProductDevelopment.mdx]: Corrected as well.

File: docs/Product/ProductDevelopment.mdx (hunk 8 above)
Comment (mdzzg) [reply docs/Product/ProductDevelopment.mdx]: Sentence case modification in the new commit of this PR.

File: docs/Product/ProductDevelopment.mdx (hunk 9 above)
Comment (mdzzg) [reply docs/Product/ProductDevelopment.mdx]: Corrected.

**Comment by mdzzg** [comment]
Hi Fabrizio & Akshat.

I believe I have addressed all of the above. In case I have missed something, which is quite possible, or further improvements are needed, kindly write me a comment, and I'll address it as quickly as possible, as the other OET tasks allow me to do so.

With many thanks for these very useful comments,
Mario

**Review by finozzifa — APPROVED** [review]

**Comment by drifter089** [comment]
merged changes in #44

**Pull request closed without merging**